src/bin/electrs.rs

```diff
@@ -80,7 +80,15 @@ fn run_server(config: Arc<Config>) -> Result<()> {
         &metrics,
         Arc::clone(&config),
     )));
-    mempool.write().unwrap().update(&daemon)?;
+    loop {
+        match Mempool::update(&mempool, &daemon) {
+            Ok(_) => break,
+            Err(e) => {
+                println!("Error performing initial mempool update, trying again in 5 seconds: {}", e);
```

Comment:
This should use `warn!()` and `display_chain()`

Reply:
Done.